### ISSUE TITLE
EHOSTUNREACH

### ISSUE THREAD

**Issue by lespatots** [merdok/homebridge-webos-tv#36]
I am getting these message on a regular basis

[2018-4-30 10:57:12] [LGwebOSTV] webOS - Error: socket hang up
[2018-4-30 10:57:12] [LGwebOSTV] webOS - Error: connect EHOSTUNREACH 192.168.1.199:3000

I can turn the TV on remotely using Homekit and when off it is in standby. After some time, between 12 and 24 hours Homebridge crashes. Unfortunately I suffer from the missing log issue so I have no more detail. Could the crash be related to this error?

**Comment by merdok** [comment]
Hi,

I don't think that the crash is related to this error. This only notifies you that the TV is no longer reachable, probably because you turned it off.

Greets

**Comment by lespatots** [comment]
Merdok,

The TV is not off as it responds to commands. Also I have further results to share. After operating the TV, the error went away and I had no crash for over 24 hours. I have just restarted Homebridge as I added a new function for the TV. The errors are back! The TV is in standby but it does not respond to a ping. So I can use the Homekit (via the plugin and Homebridge) and it turned on, responded to pings, turned it off and it responds to pings and of course the errors have stopped. Perhaps there is a weird off mode for the LG TV?

However, another strange result was that all the buttons shifted function. I added my new function "Live TV" at the beginning of the list and the the subsequent items moved down by one. Not a big issue but worthy of a note in the setup guide?
